.gitignore

```diff
@@ -58,3 +58,4 @@ dist
 !test/extend.d.ts
 !test/plugin/typography/src/*.d.ts
 !test/plugin/typography/index.html
+!example/index.html
```

Comment:
why add this?

Reply:
Since there is `*.html` in that file and that was getting ignored.

Reply:
Sorry I must have added that change before and forgotten to remove it. Thanks for noticing!